tests/rules/prefer-immediate-return.test.ts

```diff
@@ -186,6 +186,11 @@ ruleTester.run("prefer-immediate-return", rule, {
         }
       `,
       errors: [{ line: 3, column: 19, endColumn: 21 }],
+      output: `
+        function let_returned() {
+          return 42;
+        }
+      `,
```

Comment:
[Since 7.x, ESLint requires `output` property when testing rules providing an autofix](https://eslint.org/docs/user-guide/migrating-to-7.0.0#additional-validation-added-to-the-ruletester-class).